tests/x11/yast2_lan_restart.pm

```diff
@@ -42,7 +42,20 @@ sub check_network {
     $status //= 'no_restart';
     wait_still_screen;
     send_key 'alt-o';                     # OK
-    assert_screen 'yast2_closed_xterm_visible', 120;
+                                          # new: warning pops up for firewall, alt-y for assign it to zone
```

Comment:
Better add a blank line in before so that the comment is not placed in this weird location